models/questionModel.js

```diff
@@ -0,0 +1,42 @@
+import mongoose from 'mongoose';
+
+const questionSchema = new mongoose.Schema(
+   {
+      "question": { type: String, required: true },
+      "correct": [ {type: String, required: true } ], // array of correct answer ID's 
+      "answers": [ {type: String, required: true } ], // array of possible answers as strings
+      "type": { enum: ['input', 'checkout', 'radio', 'textbox']}
```

Comment:
You are missing type and required options in schema

Reply:
added